libs/user/types.ts

```diff
@@ -56,7 +56,15 @@ export interface UserMakeTransfer extends UserMakeTransaction {
   salt?: string;
 }
 
-export interface UserMakeTransformTransfer extends UserMakeTransfer {
+export interface Token {
```

Comment:
Since there is a `Token` class, can we please rename this one: eg `L2Token`, `NightfallToken`, perhaps `CustomToken`. Language to be aligned with...

![image](https://github.com/NightfallRollup/nightfall-sdk/assets/7704645/3c5a8a0e-d3dc-4a50-9df0-7020e76fbe24)

Reply:
Moved to `L2Token`